src/tcgwars/logic/impl/gen4/MysteriousTreasures.groovy

```diff
@@ -1607,15 +1611,20 @@ public enum MysteriousTreasures implements LogicCardInfo {
             energyCost P
             attackRequirement {}
             onAttack {
-              damage 0
+              my.deck.subList(0,5).select(min:0,"Choose a card to put in your hand").moveTo(hidden: true, my.hand)
```

Comment:
Are you supposed to be able to choose 0? The text implies to me that you _must_ select 1.

Reply:
Yup, should be 1 indeed. Will correct it with the build fix 👍